src/Components/Components/test/RenderTreeBuilderTest.cs

```diff
@@ -398,15 +398,15 @@ public void CannotAddAttributeAtRoot()
         }
 
         [Fact]
-        public void CannotAddEventHandlerAttributeAtRoot()
+        public void CannotDelegateAttributeAtRoot()
```

Comment:
```suggestion
        public void CannotAddDelegateAttributeAtRoot()
```

Reply:
I have a passing PR, so I'll work this into my next batch of changes.